clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/job/RunDcosJobAtomicOperation.groovy

```diff
@@ -172,7 +172,14 @@ class RunDcosJobAtomicOperation implements AtomicOperation<DeploymentResult> {
         }
 
         if (jobDescription.env) {
-          env = jobDescription.env.clone() as Map<String, String>
+          env = jobDescription.env.clone()
+          env.each { key, value ->
+            if (!(value instanceof Map)) env[key] = value.toString()
+          }
+        }
+
+        if (jobDescription.secrets) {
```

Comment:
Is this a null check or an empty check? I'm a little biased to being explicit, personally.